adoc/MAIN.release-notes.adoc

```diff
@@ -9,13 +9,29 @@ ifeval::['{release_type}' != 'public']
 !!! This is a preview release. Not to be distributed outside of SUSE !!!
 endif::[]
 
+SUSE CaaS Platform is an enterprise-ready Kubernetes-based container management solution
```

Comment:
Instead of literally writing "SUSE CaaS Platform", use `{productname}`. (Goes for the entire section.)